### PR TITLE
Make e2e-tests main application repo

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes https://github.com/PerfectFit-project/virtual-coach-issues/issues/159

- Changes naming to make all the builds refer to the 'main' build instead of e2e test
- Add `script/server` for serving the full application
- Add `scheduler` to the spun up components (it was missing)
- Fix `script/test` script (it wasn't working because docker-compose was not run in detached state)
- Update documentation (including moving the design doc from `virtual-coach-rasa`) (see https://github.com/PerfectFit-project/virtual-coach-rasa/pull/4 for related PR to remove the documentation there)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -136,5 +136,4 @@ dmypy.json
 #Mac
 .DS_Store
 
-_test_dir
```
Comment (svenvanderburg) [inline .gitignore:139]: Renamed it to 'build', which is already in .gitignore

File: script/fetch_and_build_all_images.sh [hunk 2]
```diff
@@ -1,44 +1,49 @@
-TEST_DIR='_test_dir'
```
Comment (dsmits) [inline script/fetch_and_build_all_images.sh:1]: I would add a shebang to this script that indicates that this should be run using bash:
```suggestion
#!/bin/bash
```

File: script/server [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+bash fetch_and_build_all_images.sh
```
Comment (dsmits) [inline script/server:2]: If you add the shebang to the `fetch_and_build_all_images.sh` script and make it executable you don't have to call it here with bash. I would prefer to do it like this then:
```suggestion
./fetch_and_build_all_images.sh
```

**Review by dsmits — CHANGES_REQUESTED** [review]
Looks good! Just a couple of very small remarks

**Comment by svenvanderburg** [comment]
Nice suggestions :)

**Pull request merged**